Composer/packages/client/src/plugins/api.ts

```diff
@@ -25,7 +25,7 @@ interface AuthAPI {
 
 interface PublishAPI {
   getPublishConfig?: () => PublishConfig | void;
-  startProvision?: (config: any) => void;
+  startProvision?: (config: any, arm: string, graph: string) => void;
```

Comment:
To not break existing plugins that aren't authored in composer, you either need to make arm and graph optional or add a new method (startProvision2).